react-app-drawer/my-app/src/components/MenuItem.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+import { type MenuItem } from './AppDrawer';
+
+type Props = {
+  menuItem: MenuItem;
+  isOpen: boolean;
+};
+
+export function MenuItem({ menuItem, isOpen }: Props) {
```

Comment:
It can be confusing to have a type and a function with the same name. I usually try to distinguish them by adding something to the component name, like 'Card' or 'Item' or something.